podpac/core/coordinates/coordinates.py

```diff
@@ -1501,12 +1505,152 @@ def issubset(self, other):
 
         return all(c.issubset(other) for c in self.values())
 
-    def is_stacked(self, dim):
-        if dim not in self.udims:
-            raise ValueError("Dimension {} is not in self.dims={}".format(dim, self.dims))
-        elif dim not in self.dims:
-            return True
-        return False
+    def is_stacked(self, dim): #re-wrote to be able to iterate through c.dims
+        match (dim in self.dims) + (dim in self.udims):
+            case 0: # both false
+                raise ValueError("Dimension {} is not in self.dims={}".format(dim, self.dims))
+            case 1: # one true, one false
+                return True
+            case 2: # both true
+                return False
+
+    '''
+    Calculate horizontal resolution of coordinate system.
+    Assumes horizontal coords are in degrees.
+    
+    Support different types!
+        - "nominal" <-- this is wrong but cheap to calculate. Give a 'nominal' resolution over the entire domain
+        - "summary" <-- This is still approximate, gives the mean and standard deviation to each point
+        - "full" <-- min distance between every point (dx, dy, basically)
+    '''
+    def horizontal_resolution(self, units = "metre", type="nominal"):

```

Comment:
We use the Numpy style docstrings. Please reformat your doc strings to comply with those.

Reply:
Also looks like you're not using the `black` formatting git hook... I'll help get that set up for you.